pkg/app/src/components/Forms/Proposal/form.tsx

```diff
@@ -85,9 +133,20 @@ export function Form({ currentStep, setStep }: ComponentProps) {
 				</Button>
 
 				<Button size="large" variant="contained" onClick={handleNext} disabled={checkNextButtonState()}>
-					{currentStep === 2 ? 'Save Proposal' : 'Next step'}
+					{currentStep === 2 ? 'Create Proposal' : 'Next step'}
```

Comment:
@DarkNebula0 don't you think it's better to create it as a variable at the top of the function and use the useMemo hook for that, or not necessary?

Reply:
useMemo is not necessary for this but it should actually come through the translation file. I will change it.